.github/workflows/unit_test_mysql80.yml

```diff
@@ -164,4 +164,4 @@ jobs:
         fi
 
         # print test output
-        cat output.txt
+        cat output.txt | grep -ivE '(PASS|RUN).*(Collation|Test.*Valid|TestRemoteWildcardMatches|Fuzz|tpcc|TestParse|PITR|cases|TestPlan|Super.*long)|DBClient'
```

Comment:
I'm not sure about this part, but the rest looks good. Do you see the expected behavior in launchable for the PR branch?

Reply:
I have reverted this change so that we can merge this PR without needing additional checks. I was not sure, looking at launchable's UI, how to go about checking. So rather than make a breaking change it is best to merge the other changes, which give us a significant benefit and look at more filtering later.

Reply:
I see now launchable output in the CI

one of the CI output
```
Launchable recorded tests for build 8400631701 (test session 2749801) to workspace vitess/vitess-app from 1 files:

|   Files found |   Tests found |   Tests passed |   Tests failed |   Total duration (min) |
|---------------|---------------|----------------|----------------|------------------------|
|             1 |            16 |             16 |              0 |                   3.15 |
```